polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md

```diff
@@ -29,231 +133,1298 @@ order: 1
+    title: `Check RegExp for outdated <HorizontalGrid /> component`,
+    code: String.raw`HorizontalGrid`,
+  }}
+/>
+
+</CollapsibleDetails>
+
+<CollapsibleDetails summary="💡 Migration example">
+
+```diff
+- <HorizontalGrid />
++ <InlineGrid />
 ```
-<Box[^>\w](?:[^>]|\n)*?borderRadiusEndEnd
+
+</CollapsibleDetails>
+
+### Button
+
+#### Consolidate boolean props to `variant` and `tone`
+
+Button has consolidated and deprecated the `connectedDisclosure`, `outline`, `destructive`, `primarySuccess`, `removeUnderline`, and `monochrome` props.
+
+<Code
+  code={{
+    title: 'polaris-migrator codemod',
+    className: 'language-bash',
+    code: String.raw`npx @shopify/polaris-migrator v12-react-update-button-component <path>`,
+  }}
+/>
+
+<CollapsibleDetails summary="✅ Post-migration RegExp validation">
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button connectedDisclosure /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?connectedDisclosure`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button destructive /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?destructive`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button outline /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?outline`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button monochrome /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?monochrome`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button plain /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?plain`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button primary /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?primary`,
+  }}
+/>
+
+<Code
+  code={{
+    title: `Check RegExp for outdated <Button primarySuccess /> prop`,
+    code: String.raw`<Button[^>\w](?:[^>]|\n)*?primarySuccess`,
+  }}
+/>
+
+</CollapsibleDetails>
+
+<CollapsibleDetails summary="➡️ Prop consolidation mappings">
+
+| Old variant                         | New variant                                                                                                            |
+| ----------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `plain=true`                        | `variant="plain"`                                                                                                      |
+| `primary=true`                      | `variant="primary"`                                                                                                    |
+| `primary=true` + `plain=true`       | `variant="tertiary"`                                                                                                   |
+| `monochrome=true` + `plain=true`    | `variant="monochromePlain"` \* This will be deprecated in a future release, please use a different variant if possible |
```

Comment:
```suggestion
| `monochrome=true` + `plain=true`    | `variant="monochromePlain"` \* <br/>This will be deprecated in a future release, please use a different variant if possible |
```